### PR TITLE
Identity Analytics Alert Handling Playbook for XSIAM

### PR DESCRIPTION
**Pull request by OmriItzhak**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-9193)

## Description
The new playbook for XSIAM to handle Identity Analytics alerts.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Azure-Enrichment-Remediation/ReleaseNotes/1_1_9.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+#### Playbooks
+
+##### Azure - User Investigation
+
+- Added tasks to check if the Azure Sentinel tables are available for querying.
```
Comment (altmannyarden) [inline Packs/Azure-Enrichment-Remediation/ReleaseNotes/1_1_9.md:6]: added tasks or a task?

File: Packs/CoreAlertFields/.pack-ignore [hunk 2]
```diff
@@ -11,4 +11,7 @@ ignore=IF100
 ignore=IF115
 
 [file:incidentfield-User_Risk_level.json]
-ignore=IF115
\ No newline at end of file
+ignore=IF115
+
+[file:incidentfield-Alert_Search_Results.json]
+ignore=IF100
```
Comment (altmannyarden) [inline Packs/CoreAlertFields/.pack-ignore:17]: What 'IF100' representing?

File: Packs/Azure-Enrichment-Remediation/Playbooks/playbook-Azure_-_User_Investigation.yml [hunk 3]
```diff
@@ -1275,8 +1273,182 @@ tasks:
...
+      "yes":
+      - "13"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: containsString
+          left:
+            value:
+              simple: ${AzureTables.tables.rows.[].[0]}
```
Comment (altmannyarden) [inline Packs/Azure-Enrichment-Remediation/Playbooks/playbook-Azure_-_User_Investigation.yml:1350]: if all those 3 tasks are the same. lets remove the duplicated tasks. 

the context path doesnt look right. please verify it

File: Packs/Core/ReleaseNotes/3_0_4.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+##### New: Identity Analytics - Alert Handling
+
+- New: The `Identity Analytics - Alert Handling` playbook is designed to handle Identity Analytics alerts and executes the following:
+
+Analysis:
+- Enriches the IP and the account, providing additional context and information about these indicators.
+
+Verdict:
+- Determines the appropriate verdict based on the data collected from the enrichment phase.
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/3_0_4.md:24]:
```suggestion
Determines the appropriate verdict based on the data collected from the enrichment phase.
```

File: Packs/Core/ReleaseNotes/3_0_4.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+Investigation:
+- Checks for related XDR alerts to the user by Mitre tactics to identify malicious activity.
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/3_0_4.md:27]:
```suggestion
- Checks for related Cortex XDR alerts to the user by Mitre tactics to identify malicious activity.
```

File: Packs/CoreAlertFields/ReleaseNotes/1_0_30.md [hunk 6]
```diff
@@ -0,0 +1,4 @@
+
+#### Incident Fields
+
+- New: **Alert Search Results**
```
Comment (ShirleyDenkberg) [inline Packs/CoreAlertFields/ReleaseNotes/1_0_30.md:4]:
```suggestion
New: **Alert Search Results**
```

File: Packs/Core/Playbooks/playbook-Identity_Analytics_-_Alert_Handling_README.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+| IAMRemediationType | The response on 'Cloud Credentials Rotation - Azure' sub-playbook provides the following remediation actions using MSGraph Users:<br/><br/>Reset: By entering "Reset" in the input, the playbook will execute password reset.<br/><br/>Revoke: By entering "Revoke" in the input, the playbook will revoke the user's session.<br/><br/>ALL: By entering "ALL" in the input, the playbook will execute the reset password and revoke session tasks. | Revoke | Optional |
+| AutoContainment | Whether to execute containment plan \(except isolation\) automatically.<br/>Possible values are: "True" and "False". | False | Optional |
+| UserContainment | Whether to disable the user account using the 'Containment Plan' su-playbook.<br/>Possible values are: "True" and "False". | False | Optional |
+| ClearUserSessions | Whether to clear the user's active Okta sessions using the 'Containment Plan' su-playbook.<br/>Possible values are: "True" and "False". | True | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Identity_Analytics_-_Alert_Handling_README.md:62]:
```suggestion
| UserContainment | Whether to disable the user account using the 'Containment Plan' sbu-playbook.<br/>Possible values are: "True" and "False". | False | Optional |
| ClearUserSessions | Whether to clear the user's active Okta sessions using the 'Containment Plan' sub-playbook.<br/>Possible values are: "True" and "False". | True | Optional |
```

**Comment by ShirleyDenkberg** [comment]
@altmannyarden @MosheEichler Doc review completed.

File: Packs/Azure-Enrichment-Remediation/ReleaseNotes/1_1_9.md (hunk 1 above)
Comment (OmriItzhak) [reply Packs/Azure-Enrichment-Remediation/ReleaseNotes/1_1_9.md:6]: Tasks. 
New task to query the available tables and new conditional tasks to check whether to query each table based on the output of the available table query.

File: Packs/CoreAlertFields/.pack-ignore (hunk 2 above)
Comment (OmriItzhak) [reply Packs/CoreAlertFields/.pack-ignore:17]: The word 'Alert' cannot be included in the incident field name.

File: Packs/Azure-Enrichment-Remediation/Playbooks/playbook-Azure_-_User_Investigation.yml (hunk 3 above)
Comment (OmriItzhak) [reply Packs/Azure-Enrichment-Remediation/Playbooks/playbook-Azure_-_User_Investigation.yml:1350]: Each conditional task checks another table.
I checked the context path, and it is the correct path.

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**